package.json

```diff
@@ -66,7 +66,7 @@
   },
   "homepage": "https://github.com/betterplace/betterplace-react-components",
   "peerDependencies": {
-    "react": ">=16"
```

Comment:
@iz-podpolja  Was it correct to change the peer dependency as well?

Reply:
Yes this is correct